trackers/react-native-tracker/README.md

```diff
@@ -0,0 +1,86 @@
+# Snowplow Node Tracker
+
+[![npm version][npm-image]][npm-url]
+[![License][license-image]](LICENSE)
+
+Add Snowplow Tracking to your Node.js Application with `@snowplow/node-tracker`.  
+
+## Maintainer quick start
+
+Part of the Snowplow JavaScript Tracker monorepo.  
+Build with [Node.js](https://nodejs.org/en/) (18 - 20) and [Rush](https://rushjs.io/).
+
+### Setup repository
+
+```bash
+npm install -g @microsoft/rush 
+git clone https://github.com/snowplow/snowplow-javascript-tracker.git
+rush update
+```
+
+## Package Installation
+
+With npm:
+
+```bash
+npm install @snowplow/node-tracker
```

Comment:
Should this be `@snowplow/react-native-tracker`? (A couple of others in the doc too).